node/src/components/consensus/era_supervisor/tests.rs

```diff
@@ -329,7 +331,7 @@ async fn finalize_blocks_and_switch_eras() -> Result<(), Error> {
             &mut rng,
         )?
     };
-    assert!(effects.is_empty());
```

Comment:
If the test isn't deleted, I guess this shouldn't change?  The consensus guys will know better than me, but I'd expect the previous assertion was what the test was meant to check.